src/06_ffi.md

```diff
@@ -195,8 +200,44 @@ C++) are not obvious, cf. [RFC 2195].
 Automated tools to generate bindings, such as [rust-bindgen] or
 [cbindgen], may be of help in making types consistent between C and Rust.
 
-[rust-bindgen]: https://github.com/rust-lang/rust-bindgen
-[cbindgen]: https://github.com/eqrion/cbindgen
+[rust-bindgen]: https://crates.io/crates/rust-bindgen
+[cbindgen]: https://crates.io/crates/cbindgen
+
+### Platform-dependent types
+
+When interfacing with a foreign language, like C or C++, it is often required
+to use platform-dependent types such as C's `int`, `long`, etc.
+
+In addition to `c_void` in `std::ffi` (or `core::ffi`) for `void`, the standard
+library offers portable type aliases in `std:os::raw` (ore `core::os::raw`):
+
+- `c_char` for `char` (either `i8` or `u8`),
```

Comment:
I guess we could mention that Rust does not support platforms where `sizeof(char) != 1`

Reply:
If `sizeof(char) != 1`, their cannot be any standard C compiler on it and so no LLVM and no Rustc either. While the C standard is unclear on many things, `char` is 8 bits:
https://stackoverflow.com/questions/9727465/will-a-char-always-always-always-have-8-bits 
So while great source of trivia, I am not sure a C standard discussion is particularly relevant to the Rust guide...